packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -94,6 +96,9 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 			...properties
 		}));
 
+		this.isFooterEnabled =
+			findIndex(this.columnProperties, (column: StarkTableColumnProperties) => typeof column.footerValue !== "undefined") > -1;
```

Comment:
`typeof column.footerValue !== "undefined" && column.footerValue !== ''`

To not take into account empty footers

Reply:
oki ok